libs/designsystem/src/lib/components/form-field/_form-field-inputs.shared.scss

```diff
@@ -61,8 +61,8 @@ $form-field-input-padding: utils.size('s');
   }
 
   // for the browsers that has partial support, see https://caniuse.com/#feat=css-placeholder
-  &::-webkit-input-placeholder,
-  &::-ms-input-placeholder {
+  /* stylelint-disable-next-line selector-pseudo-element-no-unknown */
+  &::input-placeholder {
```

Comment:
Removing `::-webkit-input-placeholder` is probably ok, but maybe the comment about partial support should be updated.

However, I think removing the `-ms` prefix is a mistake: There is no `::input-placeholder` pseudo element as far as I know. So the disable rule comment in line 64 should also be removed.

I'm a bit concerned, that this is another sign of stylelint being too hard on the "remove vendor specific prefixes" stuff.

One thing is to remove prefixes that are not needed. Another thing is to replace them with invalid CSS. We should try to learn how to avoid that.

Reply:
Unfortunately, I do not think stylelint is clever enough to know what property is meant in a given place, it can only identify the prefix.

But auto-prefixer should handle this, if the correct rule is used in the first place, so I'll reduce this to `::placeholder`.